bno055_stm32.h

```diff
@@ -5,7 +5,7 @@
   extern "C" {
 #endif
 
-#include "i2c.h"
+// #include "i2c.h"
```

Comment:
Is commented code here needed? Why you removed i2c?

Reply:
I can't compile the code with `i2c.h` included because there is no `i2c.h`. I don't need `i2c.h` because I can compile the code successfully without it. All needed libraries already included when the code generated by STM32CubeMX